### PR TITLE
Stat weight improvements as described in #656

### PR DESCRIPTION
**Pull request by buimichael**
See #656

### CODE REVIEW

File: src/Parser/Core/Modules/Features/SpellInfo.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ import SPELLS from 'common/SPELLS';
 
 // This only works with actual healing events; casts are not recognized.
 export default {
-  [SPELLS.LIGHTS_EMBRACE_HEALING.id]: { // Sea Star of the Depthmother
+  [SPELLS.OCEANS_EMBRACE.id]: { // Sea Star of the Depthmother
```
Comment (MartijnHols) [inline src/Parser/Core/Modules/Features/SpellInfo.js]: Wait was this wrong? are you sure? I am pretty sure I verified it working.

Comment (buimichael) [reply src/Parser/Core/Modules/Features/SpellInfo.js]: Yes, this was wrong. Light embrace is a NLC trait. Oceans Embrace is the sea star procc.

Comment (MartijnHols) [reply src/Parser/Core/Modules/Features/SpellInfo.js]: well fuck that's going to mess with holy paladin's weights

File: src/Parser/Core/Modules/Features/SpellInfo.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ export default {
     int: false,
     crit: true,
     hasteHpct: false, // static CD
+    hasteHpm: true,
```
Comment (MartijnHols) [inline src/Parser/Core/Modules/Features/SpellInfo.js]: Just making sure: are you sure about this? It seems strange

Comment (buimichael) [reply src/Parser/Core/Modules/Features/SpellInfo.js]: Pretty sure. Tick rate should be every 3 s: https://www.warcraftlogs.com/reports/f7Gg2HRbwYPDqJ1B#fight=70&type=healing&source=241&ability=242622&view=events

Also verified here https://questionablyepic.com/restodruid-trinkets/
img

**Comment by MartijnHols** [comment]
Please update the changelog before merging

**Pull request merged**